app/src/main/res/layout/fragment_nearby_parent.xml

```diff
@@ -40,7 +40,19 @@
                 android:layout_height="match_parent"
                 android:background="@android:color/transparent" />
 
-        </FrameLayout>
+            <androidx.appcompat.widget.AppCompatTextView
+              android:id="@+id/tv_attribution"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:layout_alignParentRight="true"
+              android:layout_alignParentBottom="true"
+              android:layout_margin="4dp"
+              android:text="&#169; OPENSTREETMAP \n contributors"
```

Comment:
Referring to https://github.com/commons-app/apps-android-commons/issues/3766#issuecomment-639021857 , I believe this should be `OpenStreetMap (ODBL)`

Reply:
Looks like this one is linked. So, I think we don't have to mention "(ODBL)". But the text could be tweaked a little to match "© OpenStreetMap". Also, the contrast doesn't appear good in the screenshot when compared to "mapbox" in the other end.

Reply:
@sivaraam  Mapbox is probably showing a logo if you see the borders is black and the text color is white, as we don't have a logo I choose this color which seems to be visible. Although I am open to change the color and text to whatsoever we decide. Please suggest a good color code for the same

Reply:
> Please suggest a good color code for the same

Sure. I'll let you know if I could find something better.

Reply:
Can we turn OSM logo to the similar style to Mapbox's? I mean a white text and darker_gray borders.